laia/callbacks/decode.py

```diff
@@ -1,12 +1,31 @@
 from typing import Callable, Optional, Union
 
+import numpy as np
 import pytorch_lightning as pl
 from tqdm.auto import tqdm
 
 from laia.decoders import CTCGreedyDecoder
 from laia.utils import SymbolsTable
 
 
+def compute_word_prob(symbols, hyp, prob, input_separator):
+    # compute mean confidence score and mean confidence score by word
```

Comment:
- Use `"""` docstring style here
- Please describe the output (list of confidence scores)